uber/models.py

```diff
@@ -127,6 +127,7 @@ class MultiChoice(TypeDecorator):
 
     def __init__(self, choices, **kwargs):
         self.choices = choices
+        self.choices_dict = dict(choices)
```

Comment:
Convenience for quickly determining if a choice exists, `opt in column.choices_dict` instead of iterating over `column.choices`.